Shared/Views/NewSubView.swift

```diff
@@ -129,71 +138,62 @@ fileprivate struct OptionsCell: View {
     }
 }
 
-
 fileprivate struct CyclePickerCell: View {
+    @EnvironmentObject var partialSheetManager: PartialSheetManager
```

Comment:
Doesn't need to be an EnviromentalObject. should just be a state object

Reply:
we can do it like u said if we won't use sheet .. but for now I will leave it as it is till we decide how to view it ..